frontend/bower.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "federal-prototype",
```

Comment:
it should stay federal-prototype. it took a stupid amount of work to change it to a readable name and i'm not going back